lib/services/schema-object-factory.ts

```diff
@@ -110,6 +110,15 @@ export class SchemaObjectFactory {
         type: 'string'
       };
     }
+
+    // Path parameters are always of type string.
+    if (param.in === 'path') {
+      return {
+        ...param,
+        type: 'string'
+      };
+    }
+
```

Comment:
What if I have a class that represents all path parameters and I use it as follows: `@Path() params: EndpointPathParams`. I want its properties to _be spread_ and represent multiple path parameters. Would this PR still support this case?

Reply:
Sorry for the late response, just had time to investigate: I couldn't find a decorator named `@Path` in the nestjs project. 
Did you mean `@Param`?. If that's the case we have a conflict here. The questions is, who is using this decorator that way.

Reply:
I meant `@Param()`. As for who is using this decorator that way, I suppose pretty much everyone (given that this is how it's described in the docs)